server/app/repository/ApplicationRepository.java

```diff
@@ -353,17 +353,11 @@ public void updateDraftApplicationProgram(long applicantId, long programId) {
                   queryProfileLocationBuilder.create("updateDraftApplicationProgram"))
               .findOne();
 
-      if (existingDraft == null) {
```

Comment:
should we still handle the null case?

Reply:
No. If it's null there's no match so there's no need to do the update at all.